nautobot/docs/user-guides/git-data-source.md

```diff
@@ -0,0 +1,269 @@
+# Git as a Data Source
+
+The "Git™ as a Data Source" feature was developed to provide the ability to populate existing data, templates, scripts, and much more into Nautobot; while leveraging the benefits that tools such as GitHub and GitLab already provide, including issue tracking, discussions, pipelines, and approvals.
+
+For more technical details on how to use this feature, please see the documentation on [Git Repositories](../models/extras/gitrepository.md).
+
+
+## Supported Providers
+The feature uses the concept of a `provides` field to map a repository to a use case. A list of the supported options is provided below.
+
+### Core Functionality
+|Name|Summary|
+|:--|:--|
+|[Export Templates](../models/extras/exporttemplate.md)|Nautobot allows users to define custom templates that can be used when exporting objects.|
```

Comment:
These links seem to link to markdown files in a repo.  Can you link them appropriately to RTD?

Reply:
They're relative links to other documentation pages in the same directory. RTD/Mkdocs converts them to the appropriate relative hyperlinks when rendering to HTML - see https://nautobot--184.org.readthedocs.build/en/184/user-guides/git-data-source/#core-functionality for examples